packages/flutter_tools/templates/cocoapods/Podfile-objc

```diff
@@ -4,6 +4,12 @@
 # CocoaPods analytics sends network stats synchronously affecting flutter build latency.
 ENV['COCOAPODS_DISABLE_STATS'] = 'true'
 
+project 'Runner', {
```

Comment:
If you change this, do you need to change all the existing materialized Podfiles too?

Reply:
Only if they adopt a profile configuration, and only to avoid a warning from cocoapods as far as I can tell.